hack/build-cross.sh

```diff
@@ -8,8 +8,7 @@ host_platform="$(os::build::host_platform)"
 
 # Set build tags for these binaries
 readonly OS_GOFLAGS_TAGS="include_gcs include_oss"
-readonly OS_GOFLAGS_TAGS_LINUX_PPC64LE="gssapi"
-readonly OS_GOFLAGS_TAGS_LINUX_AMD64="gssapi"
+readonly OS_GOFLAGS_TAGS_$(echo ${host_platform} | tr '[:lower:]/' '[:upper:]_')="gssapi"
```

Comment:
There are a number of places (also in `common.sh`) that use this logic to get from platform --> tag envar name. Can you factor that into a function so the logic is a little more unified?